_templates/header.php

```diff
@@ -40,21 +40,6 @@
         <meta name="author"      content="<?php echo !empty($page['author']) ? $page['author'] : $sitewide['author']; ?>">
         <meta name="theme-color" content="<?php echo !empty($page['theme-color']) ? $page['theme-color'] : $sitewide['theme-color']; ?>">
 
-        <?php
```

Comment:
This little if block here makes twitter display stuff differently, something it never does with open-graph. https://dev.twitter.com/cards/types/summary-large-image